clients/wallet/src/stellar_wallet.rs

```diff
@@ -339,16 +352,18 @@ impl StellarWallet {
 			create_payment_operation(destination_address, asset, stroop_amount, self.public_key())?
 		};
 
-		self.send_to_address(request_id, stroop_fee_per_operation, vec![payment_op])
-			.await
+		self.send_to_address(request_id, vec![payment_op]).await
 	}
 
 	pub(crate) async fn send_to_address(
 		&mut self,
 		request_id: [u8; 32],
-		stroop_fee_per_operation: u32,
 		operations: Vec<Operation>,
 	) -> Result<TransactionResponse, Error> {
+		let stroop_fee_per_operation = get_fee(self.is_public_network, FeeAttribute::default())
+			.await
+			.map_err(|e| Error::FailedToGetFee(e))?;
+
```

Comment:
I don't think it matters too much but we could move the lock above this function to avoid accidentally querying too often, right?

Reply:
moved down, under the lock.